### PR TITLE
Alpha brede

### PR DESCRIPTION
**Pull request by BredeFK**
* Fixed join class bug
* Fixed submission/review bug
* Updated faq to be more professional
* Fixed it so it's only one join course link now
* When a user updates an assignment, the `submitted` time is updated too
* Also added ```TODO : time``` on places that used time, for later refactoring

### CODE REVIEW

File: webservice/controller/admin_assignment.go [hunk 1]
```diff
@@ -404,11 +413,44 @@ func AdminUpdateAssignmentPOST(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 	}
+
 	submissionID := sql.NullInt64{
 		Int64: int64(val),
 		Valid: val != 0,
 	}
 
+	// Delete former submissions if admin changes submission form TODO brede
```
Comment (JohanAanesen) [inline webservice/controller/admin_assignment.go]: Is this still TODO?

File: webservice/controller/admin_course.go [hunk 2]
```diff
@@ -54,7 +54,7 @@ func AdminCreateCoursePOST(w http.ResponseWriter, r *http.Request) {
 	user := session.GetUserFromSession(r)
 
 	course := model.Course{
-		Hash:        xid.NewWithTime(time.Now()).String(),
+		Hash:        xid.NewWithTime(time.Now()).String(), // TODO time
```
Comment (JohanAanesen) [inline webservice/controller/admin_course.go]: this is a hash, doesn't need a time setting. remove todo

File: webservice/template/admin/course/index.tmpl [hunk 3]
```diff
@@ -1,24 +1,49 @@
 {{define "title"}}Courses{{end}}
-{{define "head"}}{{end}}
+{{define "head"}}
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```
Comment (kongebra) [inline webservice/template/admin/course/index.tmpl]: jquery and bootstrap should be in the "foot", and are in the base.tmpl

File: webservice/template/admin/index.tmpl [hunk 4]
```diff
@@ -1,5 +1,9 @@
 {{define "title"}}Dashboard{{end}}
-{{define "head"}}{{end}}
+{{define "head"}}
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (kongebra) [inline webservice/template/admin/index.tmpl]: these should be in the "foot", and are in the base.tmpl!?

File: webservice/template/admin/submission/create.tmpl [hunk 5]
```diff
@@ -9,14 +9,14 @@
...
-    });
-</script>
+    <script src="/static/js/form.js"></script>
+    <script>
+        let form = new Form();
+        form.Initialize({
+            output: '#output',
+            title: 'Create new Submission form',
+            request: '/admin/submission/create',
+            weighted: true,
```
Comment (kongebra) [inline webservice/template/admin/submission/create.tmpl]: Submissions are not weighted
`weighted: false,`

File: webservice/template/admin/course/index.tmpl (hunk 3 above)
Comment (BredeFK) [reply webservice/template/admin/course/index.tmpl]: If I move both or either onw of theese to `foot`, the tooltip won't work properly

File: webservice/template/admin/index.tmpl (hunk 4 above)
Comment (BredeFK) [reply webservice/template/admin/index.tmpl]: If I move both or either onw of theese to `foot`, the tooltip won't work properly

Comment (kongebra) [reply webservice/template/admin/index.tmpl]: This should not be done, cause you force the client to load these libs 2x times 👎

**Pull request merged**